src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -141,61 +151,77 @@ export class SearchMenu extends React.Component {
     }
 
     return (
-      <div className="vads-u-display--flex vads-u-flex-direction--column vads-u-width--full">
-        <form
-          acceptCharset="UTF-8"
-          action={this.state.searchAction}
-          id="search"
-          method="get"
-          onSubmit={this.logSearchEvent(undefined)}
-        >
-          <label htmlFor="query" className="usa-sr-only">
-            Search:
-          </label>
-
-          <div className="va-flex">
-            <input
-              autoComplete="off"
-              ref="searchField"
-              className="usagov-search-autocomplete vads-u-margin-right--0p5"
-              id="query"
-              name="query"
-              type="text"
-              onChange={this.handleInputChange}
-              value={this.state.userInput}
-            />
-
-            <button
-              type="submit"
-              disabled={!validUserInput}
-              className="vads-u-margin-left--0p25"
-            >
-              <IconSearch color="#fff" />
-              <span className="usa-sr-only">Search</span>
-            </button>
+      <Downshift
+        id="search"
+        inputValue={this.state.userInput}
+        onSelect={item => this.handleSearchEvent(item)}
+        itemToString={item => {
+          return item;
+        }}
+        isOpen={this.props.isOpen}
+      >
+        {({
```

Comment:
Weird, I didn't know we could have functions as React children lol `Downshift` is odd

Reply:
PREACH. Downshift has been a major learning experience for me

Reply:
Oh yeah...Downshift is a joy to use 😭